### PR TITLE
Update Keen Dreams and add Reflection Keen Dreams

### PR DESCRIPTION
**Pull request by pehsa**

### CODE REVIEW

File: games/r.yaml [hunk 1]
```diff
@@ -443,3 +443,14 @@
...
   updated: 2015-04-13
   url: http://www.ryzomcore.org/
+  
+- name: Reflection Keen Dreams
+  lang: C
+  license:
+  - GPL2
+  development: sporadic
+  originals:
+  - Commander Keen Series
```
Comment (cxong) [inline games/r.yaml]: consider adding catacomb 3d as an original!

File: games/r.yaml [hunk 2]
```diff
@@ -443,3 +443,14 @@
...
+  repo: https://github.com/NY00123/refkeen
+  type: remake
+  updated: 2019-03-12
```
Comment (cxong) [inline games/r.yaml]:
```suggestion
  updated: 2019-03-12
  info: Ports of Keen Dreams, Catacomb 3-D and the Catacomb Adventure Series
  status: playable
  content: commercial
  video:
    youtube: EeWk6WFVZ4U
    youtube: WxdkyQKLpJI
```

File: games/r.yaml [hunk 3]
```diff
@@ -443,3 +443,14 @@
   type: remake
   updated: 2015-04-13
   url: http://www.ryzomcore.org/
+  
+- name: Reflection Keen Dreams
```
Comment (cxong) [inline games/r.yaml]:
```suggestion
- name: Reflection Keen
```

File: games/r.yaml [hunk 4]
```diff
@@ -443,3 +443,20 @@
...
+  - Commander Keen Series
+  repo: https://github.com/NY00123/refkeen
+  type: remake
+  updated: 2019-03-12
+  info: Ports of Keen Dreams, Catacomb 3-D and the Catacomb Adventure Series
+  status: playable
+  content: commercial
+  video:
+    youtube: EeWk6WFVZ4U
+    youtube: WxdkyQKLpJI
```
Comment (cxong) [inline games/r.yaml]:
```suggestion
```
oops, I guess only one youtube is allowed

**Comment by pehsa** [comment]
@cxong Thanks for the suggestions

File: originals/c.yaml [hunk 5]
```diff
@@ -95,6 +95,26 @@
...
+  external:
+    wikipedia: Catacomb 3-D
+  meta:
+    genre:
+    - Action
+    - FPS
+    theme:
+    - Fantasy
+    
+- name: Catacomb Adventure Series
```
Comment (cxong) [inline originals/c.yaml]: If this is the same game as Catacomb 3-D, we shouldn't add it here. Instead we can have it as an alternate name to Catacomb 3-D

Comment (pehsa) [reply originals/c.yaml]: Not the same game.
The Adventure Series is the name of the trilogy ( Catacomb Abyss, Catacomb Armageddon, Catacomb Apocalypse ). The Adventure Series the sequel to the 3-D.

**Pull request merged**